Makefile

```diff
@@ -72,3 +72,14 @@ check: lint test generate_check
 go-mod-update:
 	go get -u $$(go run ./hack/allmods)
 	go mod tidy
+
+.PHONY: dockerimage
+dockerimage: build
+	docker build -t asalkeld/nitric-cli:latest .
+	docker tag asalkeld/nitric-cli:latest asalkeld/nitric-cli:x64
+	docker tag asalkeld/nitric-cli:latest asalkeld/nitric-cli:${SOURCE_GIT_COMMIT}
+	docker push asalkeld/nitric-cli:latest
+	docker push asalkeld/nitric-cli:x64
+	docker push asalkeld/nitric-cli:${SOURCE_GIT_COMMIT}
+	docker manifest create asalkeld/nitric-cli:latest --amend asalkeld/nitric-cli:x64  --amend asalkeld/nitric-cli:${SOURCE_GIT_COMMIT}
+	docker manifest push asalkeld/nitric-cli:latest
```

Comment:
Let's use github actions to publish this, I posted a sample in the last review.

Reply:
updated now @tjholm can you ptal?

Reply:
@asalkeld looks good, just one more comment/question on develop v. main publishing.